mail/mailer.go

```diff
@@ -32,12 +38,31 @@ func New(server, port, username, password string) MailerImpl {
 		Port:   port,
 		Auth:   auth,
 		From:   username,
+		clk:    clock.Default(),
+	}
+}
+
+func (m *MailerImpl) generateMessage(to []string, subject, body string) []byte {
+	now := m.clk.Now().UTC()
+	rand.Seed(int64(now.Nanosecond()))
```

Comment:
And seeding by time is a good way to make pseudo-random numbers much more predictable, btw.